client-app/shared/catalog/components/product-card-recently-browsed.vue

```diff
@@ -101,6 +101,6 @@ function onValidationUpdate(validation: { isValid: true } | { isValid: false; er
 }
 
 watch(countInCart, (newCount) => {
-  quantity.value = newCount || 1;
+  quantity.value = newCount || product.value.minQuantity || 1;
```

Comment:
was it a bug?

Reply:
it was a not related to the task bug, which I spotted during testing my changes, decided to fix it at once in that PR